Formula/slicot.rb

```diff
@@ -0,0 +1,22 @@
+class Slicot < Formula
+  desc "Fortran 77 algorithms for computations in systems and control theory"
+  homepage "http://www.slicot.org"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+  version "5.0+20101122"
+  sha256 "fa80f7c75dab6bfaca93c3b374c774fd87876f34fba969af9133eeaea5f39a3d"
+
+  depends_on "gcc"
+
+  def install
+    args = [
+      "FORTRAN=#{ENV.fc}",
```

Comment:
`FORTRAN=gfortran`
`LOADER=gfortran`

Inline those args directly into the two calls below.